AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -424,8 +424,14 @@ - (void)applyEnabledState:(BOOL)isEnabled {
 #pragma mark - MSAuthTokenContextDelegate
 
 - (void)authTokenContext:(MSAuthTokenContext *)__unused authTokenContext didUpdateAccountIdWithAuthToken:(NSString *)authToken {
+
+  // TODO: consume the unique account id once provided in authTokenContext.
```

Comment:
Is this tracked anywhere?

Reply:
yes, don't know the WI but team is aware of that

Reply:
have created work items for these